donkeycarManagerFront/src/components/dashboard.vue

```diff
@@ -383,8 +384,13 @@ export default {
     async resumeJobs (player) {
       await srv.resumeJobs(player)
     },
-    async pauseJobs (player) {
-      await srv.pauseJobs(player)
+    async reload (player) {
```

Comment:
You call the argument player but in the template you are passing a job, is really confusing, change the argument name.